src/Proj4.jl

```diff
@@ -136,6 +142,9 @@ function transform!(src::Projection, dest::Projection, position::Array{Float64,2
     position
 end
 
+transform!(src::Projection, dest::Projection, position::Vector{Float64}; radians::Bool=false) =
+    transform!(src, dest, reshape(position,(1,length(position))), radians=radians)
+
```

Comment:
Looks good.

General question: do you think specifying sets of N coordinates as Nx3 matrices makes sense in the context of other julia libraries you've used, vs the alternative of 3xN?  The option here is the way I've habitually done it in matlab, so that's what I've got here.  However, some libraries (eg, KDTrees) want the transpose which is a pain.